src/templates/next/typography/Heading.ts

```diff
@@ -2,9 +2,11 @@ export const Heading = {
   1: "text-[2.75rem] leading-[3.25rem] lg:text-[3.75rem] lg:leading-[4rem] font-semibold tracking-[-0.022em]",
   2: "text-[2.375rem] leading-[2.75rem] lg:text-[3rem] lg:leading-[3.625rem] font-semibold tracking[-0.022em]",
   3: "text-[1.625rem] leading-[2rem] lg:text-[2.25rem] lg:leading-[3rem] font-semibold tracking-[-0.022em]",
-  4: "text-[1.125rem] leading-[1.5rem] lg:text-[1.5rem] lg:leading-[2.25rem] font-semibold",
+  4: "text-[1.125rem] leading-[1.5rem] lg:text-[1.5rem] lg:leading-[2.25rem] font-semibold tracking[-0.022em]",
   "4-medium":
```

Comment:
sense checking here, does heading 5 not need tracking ah?

Reply:
nit, perhaps we can follow this convention in the future prs: if -0.022em is something that we are going to use quite often, we can declare them centrally as [such](https://tailwindcss.com/docs/letter-spacing#using-custom-values). this way we don't have a hardcoded number repeated in our codebase.

Reply:
no need for `heading-5` i think! quite small to have tight letter spacing

added token in [25ae7bc](https://github.com/isomerpages/isomer-components/pull/100/commits/25ae7bcce4237d0ba0e8a728a131c5e7e061e473)